changelogs/fragments/1345-max_rc_more_than_0_doesn_not_put_change_as_true.yml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  - zos_job_submit - job submit when mx rc was given by the user the change value return as false
```

Comment:
I don't understand this sentence "job submit when mx rc was given by the user the change value return as false
    when the max rc is not 0." could you rephrase it?  Also, this other part "Fix now return a change value of true when the rc is not 0 and
    max rc is above or equal the value the user insert." I understand that "change" value really is "changed" from the response dict right?

Reply:
Yes thanks for the changed value